src/Symfony/Component/HttpFoundation/File/MimeType/MimeTypeExtensionGuesser.php

```diff
@@ -599,6 +599,7 @@ class MimeTypeExtensionGuesser implements ExtensionGuesserInterface
         'application/x-xliff+xml' => 'xlf',
         'application/x-xpinstall' => 'xpi',
         'application/x-xz' => 'xz',
+        'application/x-zip-compressed' => 'zip',
```

Comment:
the page you linked to was also listing `application/x-zip`. Should it be added too ?

Reply:
Good point, I've only experienced x-zip-compressed, but happy to add the others if you'd prefer?